packages/components/src/Popover/types.ts

```diff
@@ -0,0 +1,21 @@
+export type Variant =
```

Comment:
Namespace to prevent duplicate types at compile

```suggestion
export type PopoverVariant =
```